src/modules/cmdpal/Exts/ProcessMonitorExtension/TerminateProcess.cs

```diff
@@ -10,10 +10,12 @@ namespace ProcessMonitorExtension;
 internal sealed partial class TerminateProcess : InvokableCommand
 {
     private readonly ProcessItem _process;
+    private readonly ProcessListPage _owner;
 
-    public TerminateProcess(ProcessItem process)
+    public TerminateProcess(ProcessItem process, ProcessListPage owner)
     {
         _process = process;
+        _owner = owner;
         Icon = new("\ue74d");
         Name = "End task";
```

Comment:
Same as last review from before rebase -- localization is needed. We decided that we'll have a separate PR for all things localization though, so OK for now.

Reply:
unrelated but why does it keep going from 20 to 18 I'm baffled. This if for "End Task" string.